newIDE/app/src/Utils/Error.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Paper from 'material-ui/Paper';
+
+// Define error component style.
+const styles = {
+    paper: {
+    }
+};
+/**
+ * Error component
+ */
+export default class Error extends React.Component {
```

Comment:
That's a good start too, but let's keep this for another Pull Request :)
It's easier to work and merge small Pull Request ;)